PWGHF/vertexingHF/AliHFSystErr.cxx

```diff
@@ -7019,6 +7020,81 @@ void AliHFSystErr::InitLctopK0S2016pPb() {
 
 }
 
+void AliHFSystErr::InitLctopK0S2017pp5TeV() {
+  //
+  // Lc->pK0S syst errors. Responsible: A. De Caro, E. Meninno
+  //  2015 pp 5 TeV sample
+  //
+  SetNameTitle("AliHFSystErr","SystErrLctopK0S2013pPb");
```

Comment:
Hi,

at this line, you need to replace "SystErrLctopK0S2013pPb" with "SystErrLctopK0S2017pp5TeV".
This is the title of the object that is written in the root output file prouced by HFPtSpectrum.C macro, so it i s useful to keep track of the method that generated it.

Ciao, Francesco